ignite/handlers/early_stopping.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+
+class EarlyStopping(object):
+    """EarlyStopping handler can be used to stop the training if no improvement after a given number of events
+
+    Args:
+        patience (int):
+            Number of events to wait if no improvement and then stop the training
+        score_function (Callable):
+            It should be a function taking a single argument, an `ignite.engines.Engine` object,
+            and return a score `float`. An improvement is considered if the score is higher.
+
+    Examples:
+    ```python
+    from ignite.engines import Engine, Events
+    from ignite.handlers import EarlyStopping
+
+    def score_function(engine):
+        val_loss = engine.state.metrics['nll']
+        return -val_loss
+
+    handler = EarlyStopping(patience=10, score_function=score_function, verbose_logger=print)
```

Comment:
we should remove the `verbose_logger` arg from the example

Reply:
Correct! Sorry for this!